specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/library.json

```diff
@@ -0,0 +1,210 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-06-01-preview",
+    "title": "SynapseManagementClient",
+    "description": "Azure Synapse Analytics APIs"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "type": "oauth2",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      },
+      "description": "Azure Active Directory OAuth2 Flow"
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/libraries": {
```

Comment:
>libraries [](start = 127, length = 9)

Is there a requirement for supporting list across multiple workspaces?

Reply:
This is not across multiple workspaces. This is only to provide all libraries in a workspace as we have to list bigDataPools and SqlPools.